### PR TITLE
Add SimplePlanes as easy

### PR DESCRIPTION
**Pull request by USBv3**
It has quite a few steps but since most of them are just "press button" i don't think it needs a medium rating. Plus most of them can be skipped by using the direct account deletion link anyways.

### CODE REVIEW

**Comment by USBv3** [comment]
What does this failed check thing mean? Is there something wrong with the PR?

File: _data/sites.json [hunk 1]
```diff
@@ -10535,6 +10535,16 @@
             "simplenote.com"
         ]
     },
+    
+    {
```
Comment (tupaschoal) [inline _data/sites.json:10539]:
```suggestion

    {
```

**Review by tupaschoal — COMMENTED** [review]
Means there are extra unnecessary whitespaces, I've pointed out where to fix them.

File: _data/sites.json (hunk 1 above)
Comment (USBv3) [reply _data/sites.json]: Fixed it.

**Comment by USBv3** [comment]
I'm closing this and opening a new pull request since i messed up the code by accident, guess changing settings on a pr repo wasn't such a great idea.

**Pull request closed without merging**